prov/sm2/src/sm2_attr.c

```diff
@@ -35,8 +36,8 @@
 #define SM2_TX_CAPS (OFI_TX_MSG_CAPS | FI_TAGGED)
 #define SM2_RX_CAPS                                                                      \
 (FI_SOURCE | OFI_RX_MSG_CAPS | FI_TAGGED | FI_DIRECTED_RECV | FI_MULTI_RECV)
-#define SM2_HMEM_TX_CAPS ((SM2_TX_CAPS))
-#define SM2_HMEM_RX_CAPS ((SM2_RX_CAPS))
+#define SM2_HMEM_TX_CAPS (0)
+#define SM2_HMEM_RX_CAPS (0)
```

Comment:
This shouldn't be in this patch and if you're going to set it to 0, why not just remove it?

Reply:
It will be added in soon and this is just a place holder.

Reply:
fixed